fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java

```diff
@@ -1459,13 +1459,9 @@ void loadTxnCommitImpl(TLoadTxnCommitRequest request, TStatus status) throws Sta
                 if (!(attachment instanceof ManualLoadTxnCommitAttachment)) {
                     break;
                 }
-                ManualLoadTxnCommitAttachment streamAttachment = (ManualLoadTxnCommitAttachment) attachment;
```

Comment:
better to move routine load metrics update to `updateTransactionMetrics`

Reply:
done